core/src/main/scala/org/apache/pekko/persistence/r2dbc/journal/mysql/MySQLJournalDao.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
this looks brand new (no code copied from an existing source file) - therefore, could you use the standard license header (like the create sql)?